tests/test_commands.py

```diff
@@ -0,0 +1,188 @@
+import os
+import signal
+import shutil
+import subprocess
+import sys
+import time
+
+import pytest
+import requests
+import sh
+
+
+FIXTURES = 'tests/fixtures'
+WEBSITE = os.path.join(FIXTURES, 'website.py')
+BUILDDIR = os.path.join(FIXTURES, '_build')
+INDEX = os.path.join(BUILDDIR, 'index.html')
+CNAME = os.path.join(BUILDDIR, 'CNAME')
+
+
+def parametrized_fixture(*args, **kwargs):
+    @pytest.fixture(scope='module', params=args + tuple(kwargs.keys()))
+    def fixture(request):
+        if request.param in kwargs:
+            return kwargs[request.param]
+        return request.param
+    return fixture
+
+
+class Cname(str):
+    '''
+    Helper class that is str, but has different truthfulness
+    '''
+    def __bool__(self):
+        return self == ' --cname'
```

Comment:
Using `bool(cname)` instead of `cname == ' --cname'` is a cute magic gimmick, but it doesn't help readibility at all.
Consider a `is_cname(option: str)` or `cname_option(enabled: bool)` function.